docs/tutorial/install-php-library.txt

```diff
@@ -9,17 +9,13 @@ Prerequisites
 
 The MongoDB PHP Library is a high-level abstraction for the
 MongoDB PHP driver. As such, you must install the `mongodb`
-extension to use the |php-library|:
+extension to use the |php-library|.
 
-.. code-block:: sh
-
-   pecl install mongodb
-   echo "extension=mongodb.so" >> `php --ini | grep "Loaded Configuration" | sed -e "s|.*:\s*||"`
-
-Instructions for installing the `mongodb` extension on HHVM may be
-found in the :php:`Installation with HHVM
-<mongodb.installation.hhvm>` article
-in the driver documentation.
+:php:`Installing the MongoDB PHP Driver <mongodb.installation.php>`
+describes how to install the MongoDB PHP Driver. Instructions for
```

Comment:
The manual appears to be a bit inconsistent with whether we capitalize "driver". Apart from `"|php-library|"`, which is replaced with "MongoDB PHP Library", I'd like to start using lowercase for "driver" and "library". I'll make the change.